requirements/base.in

```diff
@@ -1,10 +1,12 @@
 # Core requirements for using this application
 
-Flask-Dance[sqla]>=0.9,<0.10
+-c constraints.txt
+
+Flask-Dance[sqla]
```

Comment:
Since you have already defined it in `constraints.txt`, there is no need to add it here.